Project.toml

```
@@ -14,6 +14,6 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 Crayons = "4"
-HTTP = "0.8, 0.9"
+HTTP = "0.8, 0.9, 1"
```

```suggestion
HTTP = "1"
```
I think we'll need to make a hard break here with the use of `HTTP.listen!` and the new `WebSockets` constructor.

---

Yea, I thought about supporting both wifh a version check, but might be too annoying.